examples/cluster-default/02_apps.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: application.giantswarm.io/v1alpha1
+kind: App
+metadata:
+  labels:
+    app-operator.giantswarm.io/version: 0.0.0
+    giantswarm.io/managed-by: flux
+  name: cluster01
+  namespace: org-multi-project
+spec:
+  name: cluster-cloud-director
+  namespace: org-multi-project
+  version: 0.7.1
+  catalog: cluster
+  kubeConfig:
+    inCluster: true
+  userConfig:
+    configMap:
+      name: cluster01-user-values
+      namespace: org-multi-project
+---
+apiVersion: application.giantswarm.io/v1alpha1
+kind: App
+metadata:
+  labels:
+    app-operator.giantswarm.io/version: 0.0.0
+    giantswarm.io/managed-by: flux
+  name: cluster01-default-apps
```

Comment:
After moving CPI/CSI/CNI to this app, I don't think we should have default-apps-cloud-director here.

Reply:
I see what you mean, however, this is already the version with CSI/CPI/CNI embedded in the cluster chart.
The idea is to give GS users/customers a template to create a cluster from A to Z easily.
I added a readme at the root of the examples folder to give a bit of context. wdyt?
[9b19e4f](https://github.com/giantswarm/cluster-cloud-director/pull/106/commits/9b19e4fa20bc993ee1265b431355307db42b68db)